index.src.html

```diff
@@ -177,11 +178,27 @@
   [=interface/inherits=], either directly or indirectly, from the {{Credential}} interface. This
   document defines two such interfaces, {{PasswordCredential}} and {{FederatedCredential}}.
 
+  A [=credential=] is <dfn export for="credential">effective</dfn> for a particular [=origin=] if it
+  is accepted as authentication on that origin. An [=credential/effective=] [=credential=] may stop
+  being effective if, for example, the account holder changes their password or disassociates a
+  federated account.
+
+  A [=credential=] may also only be [=effective=] for a single use or a limited amount of time. For
+  example, when a private key is used to sign a challenge, that same signature won't work for a
+  future login. When a token received over SMS is used to log into a site, that token generally
+  won't work the next day.
+
+  Single-use and time-limited [=credentials=] are generated by a <dfn export>credential
+  source</dfn>, which could be a private key, access to a federated account, the ability to receive
+  SMS messages at a particular phone number, or something else. [=Credential sources=] are not
+  exposed to Javascript or explicitly represented in this specification.
```

Comment:
This seems like a pretty reasonable distinction. Thanks for putting it together.

I think I'd like to see some discussion of how we know when a credential is effective. That is, I _think_ the user agent can only only judge efficacy as a result of trying to validate the credential. So we might know that a credential _was_ effective, a posteriori, but can't make a priori judgments about whether a credential _is_ effective.